platformio.ini

```
@@ -0,0 +1,42 @@
+[env:EncoderTest]
+platform = raspberrypi
```

```suggestion
platform = https://github.com/maxgerhardt/platform-raspberrypi.git
board_build.core = earlephilhower
```
For this one and the others